.gitignore

```diff
@@ -44,7 +44,6 @@ confdefs.h
 conftest.cpp
 conftest.err
 
-cache/
```

Comment:
I don't think that we want these changes; local builds will populate other elements in the cache directory.

Reply:
I did this on a branch where I had run builds, and it didn’t seem to be ignoring anything.

Here are the cache files I could find:
* `depends/x86_64-pc-linux-gnu/vendored-sources/wasm-bindgen/src/cache/` – ignored by depends/.gitignore
* `qa/cache/` – has its own entry in .gitignore
* `qa/rpc-tests/cache/` – we don’t want to ignore

The problem is that `cache/` is too broad of an ignore. If there’s a top-level cache that we do want to ignore, then we need `/cache/`. But I didn’t find one or anything else I could specialize this ignore to, so I figured better to remove it and add the specific case when it’s encountered. Un-anchored ignores should be for things where they can be dynamically introduced fairly broadly (e.g., `*.o`). Each cache should probably have a specific ignore entry.

Reply:
Aha, okay, I get it now.